Doc/library/tempfile.rst

```diff
@@ -404,13 +404,13 @@ Here are some examples of typical usage of the :mod:`tempfile` module::
 
     # create a temporary file using a context manager
     # close the file, use the name to open the file again
-    >>> with tempfile.TemporaryFile(delete_on_close=False) as fp:
-    ...    fp.write(b'Hello world!')
-    ...    fp.close()
-    # the file is closed, but not removed
-    # open the file again by using its name
-    ...    with open(fp.name) as f
-    ...        f.read()
+    >>> with tempfile.NamedTemporaryFile(delete_on_close=False) as fp:
+    ...     fp.write(b'Hello world!')
+    ...     fp.close()
+    ... # the file is closed, but not removed
+    ... # open the file again by using its name
+    ...     with open(fp.name) as f:
```

Comment:
Shouldn't this have a mode argument of `rb` to get `b'Hello world!'` from `f.read()`?

Reply:
The default mode is `w+b`, which should work just fine.

Reply:
> For `builtins.open`? I think not :)

True. I focused on the `NamedTemporaryFile` and missed the `open` below 🤦‍♂️